### ISSUE TITLE
Warning does not appear in the case of one-line method.

### ISSUE THREAD

**Issue by teaplanet** [checkstyle/checkstyle#588]
Warning does not appear in the case of one-line method.
Is this specification? or missed check?

```Java
public void foo() {
	bar();
}

public void bar() {
	String bar = "bar";
	bar.isEmpty();
}
```

bar method has warning of Javadoc.
But, foo method doesn't have warning.

**Comment by romani** [comment]
I do not know Check that do that validation. Is it request for new Check for checkstyle ?

**Comment by teaplanet** [comment]
Yes, it it.
I think, it is needs in the some case.
Also, I think it is not needs, but I think "I want to write meaning for only word.".

```Java
/**
 * some comments.
 *  @return zip code with dash.
 */
public String getZipCode() {
	// do something.
	return zipCode;  // 123-4567
}

/**
 * Nearly same as getZipCode, but without dash only number.
 *  @return zip code without dash.
 */
public String getZipCodeWithoutDash() {
	return getZipCode().replace("-", "");  // 1234567
}
```

You think to need a comment, aren't you?

**Comment by romani** [comment]
I expect a lot of false-positives detected by this Check, as it conflicts with Adapter pattern.

http://checkstyle.sourceforge.net/contributing.html#Introduction
ATTENTION: any new Check that you are going to introduce you need to contribute to our sanbox project sevntu.checkstyle. We are receiving only bug fixes for now to our main project, here is a reason. 

So please switch to mail-list for discussion of this idea, as it is far from ready to implement. I do not reject your idea and do think it matter but we need to find good point of where we will put violation, too much false positives will force users to switch your check off the config.

**Comment by teaplanet** [comment]
I got it.
But, I think not conflicts with Adapter pattern.
That is alias for my sense.

I'm glad to know your thinking, and I clearly.
I tried to use this leave.

If you think don't need, this issue change to close rather than incomplete.

Thanks, a lot.

**Comment by romani** [comment]
I marked it as incomplete just to stress that there no strict steps of what to implement for now.
Lets switch to mail-list for more detailed discussion of your idea - https://groups.google.com/forum/#!forum/checkstyle-devel.

I am closing issue for now, we will do reopen after we agree on implementation in mail-list.
